UNRELEASED_CHANGELOG.md

```diff
@@ -73,10 +73,29 @@
 
 ## stream-chat-android-ui-components
 ### 🐞 Fixed
-
+Fix for position of deleted messages for other users
 ### ⬆️ Improved
 
 ### ✅ Added
+Now it is possible to customize when the avatar appears in the conversation. It is
+possible to use an avatar in messages from other users and for messages of the current user. Use
+`setShowAvatarPredicate` to change the default behaviour. Default: The avatar shows only
+for messages of other users, in the bottom of a group of messages (Position.Bottom).
+Also now it is possible to apply a custom margin to messages of MessageListView. You need
+to add margins to avoid overlap between messages and avatars.
+
+If you set a predicate that shows avatars for your own messages as well, use this value:
+
+```
+streamUiMessageEndMargin=">@dimen/stream_ui_message_viewholder_avatar_missing_margin"
+```
+
+If your predicate doesn't show avatars for your own messages (this is the default behavior), remove the end margin:
+
+```
+streamUiMessageEndMargin="0dp"
+```
+
```

Comment:
- We use bullet points for items in the changelog
  - In this case, we could probably just link to the newly added docs for details instead of duplicating content